php/admin_page.php

```diff
@@ -1,4 +1,32 @@
 <?php
+require_once 'functions.php';
+require_once '../db/db_query.php';
+
+$db = getDbConnection();
+$newParagraph = $_POST['edit_form'];
+$addSubmit = $_POST['add_form'];
+$paragraphs = addAboutMe($db);
+$editDropdownResults = editParagraphDropdown($paragraphs);
+
+
+if (isset($_POST['edit'])){
+    $editChoice = $_POST['editSelect'];
+    $getEdit = getEdit($db, $editChoice);
+    $pasteEdit = pasteEdit($getEdit);
+}
+
+
+if (isset($_POST['submit'])){
+    $submitChoice = $_POST['editId'];
+    editParagraph($db, $submitChoice, $newParagraph);
+    header('Location: admin_page.php');
+}
+
+if (isset($_POST['delete'])){
+    $deleteChoice = $_POST['deleteSelect'];
+    deleteParagraph($db, $deleteChoice);
+    header('Location: admin_page.php');
```

Comment:
Likewise, do you really need this redirect? Seems a bit redundant?

Reply:
deleted it